services/src/main/java/io/druid/cli/InsertSegment.java

```diff
@@ -71,6 +74,9 @@ public InsertSegment()
   protected List<? extends Module> getModules()
   {
     return ImmutableList.<Module>of(
+        new DruidProcessingModule(),
```

Comment:
I don't know, but since `CreateTables` and `DumpSegment` weirdly depend on this stuff, and we don't test `InsertSegment` at all (neither from unit tests nor integration tests), I decided to include everything, not to break it occasionally. The same for `ResetCluster`, `ValidateSegments` and `DruidJsonValidator`.